library/src/main/java/com/android/volley/toolbox/ImageRequest.java

```diff
@@ -261,4 +261,8 @@ public void setTtl(long ttl) {
         mTtl = ttl;
         setOfflineCache(ttl > 0);
     }
+
+    public boolean isImageRequest() {
+        return true;
```

Comment:
I don't know about that. IMHO the `Request` class has nothing to do with image requests. In fact it makes no sense to have a method `Request.isImageRequest`. You're only adding that because you need `ImageRequest` to override (it's also missing `@Override`) and return true. It's just bad design and the kind of stuff that makes this library grow out of control and untested.